pkg/virt-launcher/virtwrap/network/dhcp/dhcp.go

```diff
@@ -42,6 +45,10 @@ func SingleClientDHCPServer(
 	dnsIP net.IP) error {
 
 	log.Log.Info("Starting SingleClientDHCPServer")
+	hostname, err := os.Hostname()
```

Comment:
can Pods/VMs in the same namespace contact each other with this hostname?

Ideally, whatever hostname we give a VM should be something other VM's in the same network can use to contact the VM.

Reply:
In general yes. See https://github.com/kubevirt/kubevirt/pull/775#discussion_r171779954 for more details. The only issue is that the subdomain is not added to the resolv.conf search paths, so we would either have to name our vms "vmname.vms" or not use the subdomain.

Reply:
i'm fine with using vmname.vms  approach as the hostname

Reply:
oops, i see why this is a problem though.